packages/katello/katello/katello.spec

```
@@ -8,7 +8,7 @@
 %global release 2
```

You need to reset this to 1 because of:
```
[2025-02-20T16:46:53.169Z]     changelog_epoch_version_release: 4.17.0-0.1.master
[2025-02-20T16:46:53.169Z]     msg: changelog entry missing for 4.17.0-0.2.master
[2025-02-20T16:46:53.169Z]     specfile_epoch_version_release: 4.17.0-0.2.master
```

```suggestion
%global release 1
```